templates/store/publisher-details.html

```diff
@@ -0,0 +1,139 @@
+{% extends webapp_config['LAYOUT'] %}
+
+{% block meta_copydoc %}{% endblock %}
+
+{# TODO: title, meta, etc #}
+{% block meta_title %}...{% endblock %}
+
+{% block meta_description %}...{% endblock %}
+
+{% if icon_url %}
+  {% block meta_image %}
+    {{ icon_url }}
+  {% endblock %}
+{% endif %}
+
```

Comment:
```
<script type="application/ld+json">
    {
    "@context": "http://schema.org/",
    "@id": "https://snapcraft.io/#organization",
    "@type": "Organization",
    "name": "Canonical",
    "logo": "https://assets.ubuntu.com/v1/b3b72cb2-canonical-logo-166.png",
    "url": "https://canonical.com",
    "sameAs": [
        "https://en.wikipedia.org/wiki/Canonical_(company)",
        "https://github.com/snapcore/snapcraft",
        "https://twitter.com/Canonical",
        "https://www.facebook.com/Canonical-125818784107695/",
        "https://www.linkedin.com/company/canonical-ltd-"
    ]
    }
</script>
```

This is a proposal for the schema to use on publisher pages at an organization level, it could be tricky as some publishers might not be an organization but maybe that is something we can figure out once we start implementing the public publisher pages programmatically.